dali/test/python/test_operator_cast.py

```diff
@@ -119,3 +161,15 @@ def test_operator_cast():
                 ndim = rng.integers(0, 4)
                 batch_size = rng.integers(1, 11)
                 yield _test_operator_cast, ndim, batch_size, in_type, out_type, device
+
+
+def test_operator_cast_empty_volumes():
+    types = [np.uint8, np.int32, np.float32]
+    for device in ['cpu', 'gpu']:
+        for in_type in types:
+            for out_type in types:
+                ndim = rng.integers(0, 4)
+                batch_size = rng.integers(12, 64)
+                for empty_volume_policy in ["left", "right", "middle", "all"]:
+                    yield (_test_operator_cast, ndim, batch_size, in_type, out_type, device,
```

Comment:
no need to check all combinations - we do have plenty of cases, we can do it round-robin (except `all`, which should be dirt cheap and would invalidate the test).